src/Tools/ExternalAccess/CustomMessage/External/DocumentLinePosition.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CustomMessageHandler;
+
+public struct DocumentLinePosition
```

Comment:
does this need to be defined in roslyn?  IIRC should essentially be opaque to us (part of their serialized message)?

Reply:
It doesn't strictly need to be defined in Roslyn. But defining it somewhere else, adds an additional assembly that handlers need to reference and package. So, as long as we don't need more utility methods/types for custom handlers, it simplifies the publishing story to have it defined here.

Reply:
Addressed in chat